cmd/metrictank/metrictank.go

```diff
@@ -171,7 +171,8 @@ func main() {
 	}
 
 	log.Info("Metrictank starting. Built from %s - Go version %s", gitHash, runtime.Version())
-
+	mtVersion := stats.NewBool(fmt.Sprintf("version.%s", strings.Replace(gitHash, ".", "_", -1)))
+	mtVersion.Set(true)
```

Comment:
why is this in the clustering section? maybe move it to "collect stats" section?

Reply:
the metric is deliberately set at the same time that we log the version running.  Are you suggesting that the log message should also be moved?  
In my opinion, they should be together as they both report the version being used.